packages/repack/src/utils/__tests__/getModulePaths.test.ts

```diff
@@ -0,0 +1,59 @@
+import { getModulePaths } from '../getModulePaths';
+
+describe('getModulePaths', () => {
+  const packages = [
+    'react-native',
+    'react-native-windows',
+    'react-native-macos',
+    'react-native-tvos',
+    '@callstack/react-native-visionos',
+    '@react-native/core',
+    '@babel/core',
+    '@types/react',
+    '@expo/vector-icons',
+    'socket.io',
+  ];
+
+  const paths = getModulePaths(packages);
+
+  // Classic paths are even-indexed
+  const classicPaths = paths.filter((_, index) => index % 2 === 0);
+  // Exotic paths are odd-indexed
+  const exoticPaths = paths.filter((_, index) => index % 2 !== 0);
+
+  packages.forEach((packageName, index) => {
```

Comment:
I had migrated to `it.each` but had to rollback because it doesn't provide an `index` of the iterated element. We need it to determine if the current iterated element is classic or exotic, to be able to test it correctly

Reply:
hmmm, maybe lets just call `getModulePaths` with an array with 1 element (package) in each test and then we know that 0 will have classicPath and 1 will have exoticPath? If there is a new case in the future then we can just use 2 as next one